packages/contracts/contracts/plugin/examples/count/CounterV1PluginManager.sol

```diff
@@ -18,87 +24,65 @@ contract CounterV1PluginManager is PluginManager {
         counterBase = new CounterV1();
     }
 
-    function deploy(address dao, bytes memory data)
-        external
-        virtual
+    function _getInstallInstruction(PluginManagerLib.Data memory installation)
```

Comment:
Very minor, but I would like developers to have to override a more friendly function.

Reply:
Additionally, this technically serves the purpose now, but as a future example I would make initialization much cleaner and visually intuitive of what each piece/step is.